tests/data/README.md

```diff
@@ -0,0 +1,76 @@
+# Test images
+
+## Still images
+
+### Files `kodim*`
+
+License: released by the Eastman Kodak Company for unrestricted usage
+
+Source: http://r0k.us/graphics/kodak/
+
+### Files `cosmos*`
+
+License: [Creative Commons Attribution license (reuse allowed)](https://creativecommons.org/licenses/by/3.0/legalcode)
+
+Source: ["Cosmos Laundromat" movie](https://www.youtube.com/watch?v=Y-rmzh0PI3c) (frame at around 1 minute past the
+beginning)
+
+## Metadata
+
+The structure can be displayed using `exiv2 -pS <file>`.
+
+### File `paris_exif_xmp_icc.jpg`
```

Comment:
these could be local links, but that may only make sense if the generated file is output to this directory

Reply:
I used local links and also inline images.